spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
ok, so it's not related to actual testing/specs but for coverage analysis so I would suggest to create a specific task for eager loading stuff and have the setup-simplecov task depend on that other eager loading task.